defaults/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 foreman_scap_client_state: present
 foreman_scap_client_package: ''
+foreman_scap_client_obsolete: true
```

Comment:
Should we even allow the users to make this choice?

Reply:
I've changed to be `false` by default, so we force the new client installations.

I'd leave the possibility to use the old package just in case while we mark that as deprecated.